utils/check_all_upgrades.sh

```
@@ -182,21 +181,30 @@ compatible_upgrade()
   fi
 }
 
+run_check()
+{
+  if command -v unbuffer >/dev/null 2>&1; then
+    unbuffer ${MAKE} -C ${REGDIR} check "TESTS=${SRCDIR}/regress/core/regress.sql" ${MAKE_ARGS}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)